plugins/inputs/redis_sentinel/redis_sentinel.go

```diff
@@ -0,0 +1,609 @@
+	"clients":                         "integer", // Renamed field
+	"evicted_keys":                    "integer",
+	"expired_keys":                    "integer",
+	"expired_stale_perc":              "float",
+	"expired_time_cap_reached_count":  "integer",
+	"instantaneous_input_kbps":        "float",
+	"instantaneous_ops_per_sec":       "integer",
+	"instantaneous_output_kbps":       "float",
+	"keyspace_hits":                   "integer",
+	"keyspace_misses":                 "integer",
+	"latest_fork_usec":                "integer",
+	"lru_clock":                       "integer",
+	"migrate_cached_sockets":          "integer",
+	"pubsub_channels":                 "integer",
+	"pubsub_patterns":                 "integer",
+	"redis_version":                   "string",
+	"rejected_connections":            "integer",
+	"sentinel_masters":                "integer",
+	"sentinel_running_scripts":        "integer",
+	"sentinel_scripts_queue_length":   "integer",
+	"sentinel_simulate_failure_flags": "integer",
+	"sentinel_tilt":                   "integer",
+	"slave_expires_tracked_keys":      "integer",
+	"sync_full":                       "integer",
+	"sync_partial_err":                "integer",
+	"sync_partial_ok":                 "integer",
+	"total_commands_processed":        "integer",
+	"total_connections_received":      "integer",
+	"total_net_input_bytes":           "integer",
+	"total_net_output_bytes":          "integer",
+	"used_cpu_sys":                    "float",
+	"used_cpu_sys_children":           "float",
+	"used_cpu_user":                   "float",
+	"used_cpu_user_children":          "float",
+}
+
+// Supported fields for "redis_sentinel_sentinels"
+var measurementSentinelsFields = map[string]string{
+	"down_after_milliseconds": "integer",
+	"flags":                   "string",
+	"last_hello_message":      "integer",
+	"last_ok_ping_reply":      "integer",
+	"last_ping_reply":         "integer",
+	"last_ping_sent":          "integer",
+	"link_pending_commands":   "integer",
+	"link_refcount":           "integer",
+	"name":                    "string",
+	"voted_leader":            "string",
+	"voted_leader_epoch":      "integer",
+}
+
+// Supported fields for "redis_sentinel_replicas"
+var measurementReplicasFields = map[string]string{
+	"down_after_milliseconds": "integer",
+	"flags":                   "string",
+	"info_refresh":            "integer",
+	"last_ok_ping_reply":      "integer",
+	"last_ping_reply":         "integer",
+	"last_ping_sent":          "integer",
+	"link_pending_commands":   "integer",
+	"link_refcount":           "integer",
+	"master_host":             "string",
+	"master_link_down_time":   "integer",
+	"master_link_status":      "string",
+	"master_port":             "integer",
+	"name":                    "string",
+	"role_reported":           "string",
+	"role_reported_time":      "integer",
+	"slave_priority":          "integer",
+	"slave_repl_offset":       "integer",
+}
```

Comment:
Can you please move those definitions to a `redis_sentinel_types.go` file? This makes the present file more readable IMO.

Reply:
@srebhan Addressed in https://github.com/influxdata/telegraf/pull/10042/commits/72af1414cca31f503292680d25720f2f0cb14e3e